frontend/src/index.jade

```diff
@@ -123,7 +123,10 @@ html
         .title
           .repoName {{ info.repo }}
           .author {{ info.name }} ({{ info.author }})
-          .contribution(v-if="isLoaded") Total contribution: {{ totalLineCount }} lines
+          .contribution(v-if="isLoaded")
+            span Total LoC contributed:&nbsp;
+            span.loc {{ totalLineCount }}
+          .contribution.fileType(v-if="isLoaded && totalLineCount > 0") &rarr; {{ filesLineInfo }}
```

Comment:
do this. cleaner

```
.contribution(v-if="isLoaded")
    span Total LoC contributed:&nbsp;
    span.loc {{ totalLineCount }}
    span.fileType(v-if="totalLineCount > 0") &rarr; {{ filesLineInfo }}
```

Reply:
@ongspxm i tried using v-for initially but it printed each filetype result in a separate line on the code panel, therefore i returned it as a string instead

Reply:
@ongspxm requested changes have been made, i chose to do the string generation myself for 2 reasons: 1) using v-for caused each filetype and count to be printed on separate lines 2) the elements needed to be comma-separated so i thought it is better to pass it as a string instead